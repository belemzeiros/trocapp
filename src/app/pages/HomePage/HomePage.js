import React from 'react';
import PropTypes from 'prop-types';

import { Helmet } from 'react-helmet';

import Grid from '@material-ui/core/Grid';
import { withStyles } from '@material-ui/core/styles';
// import CardPage from '../../components/CardPage';
// import CardPage from '@/components/CardPage';
import List from '../../components/List';

const styles = theme => ({
  root: {
    maxWidth: '90rem',
    marginLeft: 'auto',
    marginRight: 'auto',
    [theme.breakpoints.up('sm')]: {
      paddingTop: '16px',
      paddingLeft: '5%',
      paddingRight: '5%',
    },
  },
  box: {
    padding: '8px',
  },
  menuWrapper: {
    display: 'flex',
  },
  hidden: {
    [theme.breakpoints.down('sm')]: {
      display: 'none',
    },
  },
});

const produtos = [
  {
    id: 1,
    title: 'Item 1',
    description: 'Descrição do Item 1',
    image: 'sofa.jpg',
  },
  {
    id: 2,
    title: 'Item 2',
    description: 'Descrição do Item 2',
    image: 'sofa.jpg',
  },
  {
    id: 3,
    title: 'Item 3',
    description: 'Descrição do Item 3',
    image: 'sofa.jpg',
  },
  {
    id: 4,
    title: 'Item 4',
    description: 'Descrição do Item 4',
    image: 'sofa.jpg',
  },
  {
    id: 5,
    title: 'Item 5',
    description: 'Descrição do Item 5',
    image: 'sofa.jpg',
  },
  {
    id: 6,
    title: 'Item 6',
    description: 'Descrição do Item 6',
    image: 'sofa.jpg',
  },
  {
    id: 7,
    title: 'Item 7',
    description: 'Descrição do Item 7',
    image: 'sofa.jpg',
  },
  {
    id: 8,
    title: 'Item 8',
    description: 'Descrição do Item 8',
    image: 'sofa.jpg',
  },
];

class HomePageComponent extends React.Component {
  render() {
    const { classes } = this.props;

    return (
      <React.Fragment>
        <Helmet />
        <Grid container className={classes.root}>
          <Grid item xs={12} md={12} className={classes.box}>
            <div>Página Inicial Trocapp</div>
            <List produtos={produtos} />
          </Grid>
        </Grid>
      </React.Fragment>
    );
  }
}

HomePageComponent.propTypes = {
  classes: PropTypes.shape({
    root: PropTypes.string.isRequired,
    box: PropTypes.string,
    menuWrapper: PropTypes.string,
  }).isRequired,
};

export default withStyles(styles)(HomePageComponent);
