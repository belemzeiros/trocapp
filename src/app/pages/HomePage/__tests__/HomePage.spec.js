import React from 'react';
import TestRenderer from 'react-test-renderer';

import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

import MockDate from 'mockdate';
import { fromJS } from 'immutable';

import HomePage from '../HomePage';

describe('HomePage', () => {
  const mockStore = configureStore();

  let store;

  beforeEach(() => {
    store = mockStore(fromJS({}));
  });

  beforeAll(() => {
    MockDate.set('2020-07-01T00:00:00.000Z');
  });

  afterAll(() => {
    MockDate.reset();
  });

  it('HomePage with props will mount correctly', () => {
    const wrapper = TestRenderer.create(
      <Provider store={store}>
        <HomePage />
      </Provider>
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
