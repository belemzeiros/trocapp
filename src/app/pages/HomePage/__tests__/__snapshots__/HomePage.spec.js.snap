// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage HomePage with props will mount correctly 1`] = `
<div
  className="MuiGrid-container-5 HomePageComponent-root-1"
>
  <div
    className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-12-73 HomePageComponent-box-2"
  >
    <div>
      Página Inicial Trocapp
    </div>
    <section
      className="List-listSection-102"
    >
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 1"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 1
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 1
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 2"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 2
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 2
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 3"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 3
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 3
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 4"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 4
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 4
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 5"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 5
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 5
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 6"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 6
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 6
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 7"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 7
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 7
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
      <div
        className="MuiPaper-root-110 MuiPaper-elevation1-113 MuiPaper-rounded-111 MuiCard-root-109 CardPageComponent-card-105"
      >
        <div
          className="MuiGrid-container-5 CardPageComponent-root-104"
        >
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 CardPageComponent-card-105"
          >
            <div
              className="MuiCardMedia-root-137 CardPageComponent-media-106"
              style={
                Object {
                  "backgroundImage": "url(\\"/static/images/product/sofa.jpg\\")",
                }
              }
              title="Item 8"
            />
            <div
              className="MuiCardContent-root-139"
            >
              <h2
                className="MuiTypography-root-140 MuiTypography-headline-145 MuiTypography-gutterBottom-157 CardPageComponent-title-108"
              >
                Item 8
              </h2>
              <p
                className="MuiTypography-root-140 MuiTypography-body1-149"
              >
                Descrição do Item 8
              </p>
            </div>
            <div
              className="MuiCardActions-root-165"
            >
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Trocar
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
              <button
                className="MuiButtonBase-root-191 MuiButton-root-167 MuiButton-text-169 MuiButton-textPrimary-170 MuiButton-flat-172 MuiButton-flatPrimary-173 MuiButton-sizeSmall-188 cardpage-btn-eu-quero MuiCardActions-action-166"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex="0"
                type="button"
              >
                <span
                  className="MuiButton-label-168"
                >
                  Eu quero
                </span>
                <span
                  className="MuiTouchRipple-root-194"
                />
              </button>
            </div>
          </div>
          <div
            className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-3-64 CardPageComponent-card-105 CardPageComponent-root-104"
          />
        </div>
      </div>
    </section>
  </div>
</div>
`;
